### DISCUSSION TITLE
Should we change the Discussion top link destination?

### DISCUSSION THREAD

**Discussion by pauby** [chocolatey/boxstarter.org#4] [Ideas]
The top level 'Discussion' link points to Google Groups. I really feel that we should perhaps point this to the Discord:

* chat
* questions
* create a boxstarter channel (I'd rather avoid that as its low traffic (using Gitter as a measure) and it's yet another channel)

Thoughts @flcdrg @gep13 @mwallner ?

**Comment by gep13** [comment]
Yes, I actually had the same thought when pulling over the pages.  We should definitely look to move away from the Google Group, and I would suggest either the Discord Channel, or perhaps enable Discussions on this repository.

**Reply by pauby** [reply]
You mean discussion over on the https://github.com/chocolatey/boxstarter repository (which I see have already been enabled)?

**Reply by gep13** [reply]
Yes, exactly that.

**Reply by pauby** [reply]
I'm torn on which one to use.

**Reply by gep13** [reply]
Personally, I would encourage folks to use Discussions.  But a channel in Discord could be useful as well.

**Reply by mwallner** [reply]
I'd suggest pointing to the Chocolatey Discord channels as Paul suggested and (also as suggested) not create a new one.
Also, having discussions enabled on GitHub and pointing people there primarily is more likely a better long-term option (thinking about archive / search options on GitHub, cross-linking issues etc.).

**Comment by pauby** [comment]
I see the [Discussions link has been pointed to Discord](https://github.com/chocolatey/boxstarter.org/blob/master/input/_SiteLayout.cshtml#L61). I have actually changed my mind on this and agree with @mwallner on it being pointed here for long-term archiving. It also makes it more consistent that issues and discussions are in one place.

However, what are the thoughts? Should we leave it?

**Reply by gep13** [reply]
I am not tied to it being Discord. I updated it to point there as I didn't want the "new" site pointing at the old Google Groups mailing list.  Happy to switch this to point to the Discussions here.  I actually forgot about this particular discussion on the topic 😄
